### PR TITLE
Fix #73

### LINKED ISSUE #73: Documentation : gh-pages doesn't point on index.html
Hi.

https://sebrut.github.io/pygrocy/ needs a change in Travis file.

Best regards

### PR DESCRIPTION
**Pull request by BlueBlueBlob**
## Description

Documentation should be available at https://sebrutgithub.io/pygrocy/

**Related issue (if applicable):** fixes #73 

## Checklist
  - [x] The code change is tested and works locally.
  - [x] tests pass. **Your PR won't be merged unless tests pass**
  - [x] There is no commented out code in this PR

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ install:
 script:
   - coverage run --source=pygrocy setup.py test
   - set -e
-  - pdoc --html pygrocy
+  - pdoc3 --html pygrocy
```
Comment (SebRut) [inline .travis.yml]: is this needed?

Comment (BlueBlueBlob) [reply .travis.yml]: I don't why but with pdoc an error occurs about --html.
Maybe because it's python 3.x project.

**Comment by BlueBlueBlob** [comment]
> could you also fix the path in the README?

Well the documentation path is in your project settings.
Can you share it please ?

**Comment by BlueBlueBlob** [comment]
Ok sorry. I did this edit trough website...
I will do a new one

**Pull request closed without merging**